packages/tests/tests/store/utils.ts

```diff
@@ -148,3 +148,23 @@ export const adjustDate = (baseDate: Date, adjustMs: number): Date => {
   adjusted.setTime(adjusted.getTime() + adjustMs);
   return adjusted;
 };
+
+export const singleShardInfosToShardInfo = (
```

Comment:
nit: is this function similar to the one from `@waku/utils`? curious if we can reuse ti